Source/SwiftLintFramework/Rules/RuleConfigurations/TrailingWhitespaceConfiguration.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  TrailingWhitespaceConfiguration.swift
+//  SwiftLint
+//
+//  Created by Reimar Twelker on 12.04.16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import Foundation
+
+public struct TrailingWhitespaceConfiguration: RuleConfiguration, Equatable {
+    var severityConfiguration = SeverityConfiguration(.Warning)
+    var ignoresEmptyLines = false
+
+    public var consoleDescription: String {
+        return "ignores empty lines: \(ignoresEmptyLines ? "true" : "false")"
+    }
+
+    public init() {
+        self.init(ignoresEmptyLines: false)
+    }
+
+    public init(ignoresEmptyLines: Bool) {
+        self.ignoresEmptyLines = ignoresEmptyLines
+    }
+
+    public mutating func applyConfiguration(configuration: AnyObject) throws {
+        guard let configuration = configuration as? [String: AnyObject] else {
+            throw ConfigurationError.UnknownConfiguration
+        }
+
+        if let ignoresEmptyLinesString = configuration["ignores_empty_lines"] as? String {
```

Comment:
Why is this `String`?
Do we need additional state other than `true` or `false`?